src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.tsx.snap

```diff
@@ -891,14 +959,48 @@ exports[`Anchor renders size specific theming 1`] = `
         style="display: inline-flex;"
       >
         <svg
-          color="#000000"
-        />
-        <span
+          aria-label="LinkNext"
           class="c18"
+          viewBox="0 0 24 24"
+        >
+          <path
+            d="M2 12h20m-9-9 9 9-9 9"
+            fill="none"
+            stroke="#000"
+            stroke-width="2"
+          />
+        </svg>
+        <span
+          class="c19"
         />
         Anchor with icon
       </span>
     </a>
+    <a
+      class="c8"
+    >
+      <span
+        class="c17"
+        style="display: inline-flex;"
+      >
+        <svg
+          aria-label="LinkNext"
+          class="c20"
+          viewBox="0 0 24 24"
+        >
+          <path
+            d="M2 12h20m-9-9 9 9-9 9"
+            fill="none"
+            stroke="#000"
```

Comment:
shouldn't this be the brand color?

Reply:
The way that the color is adjusted is via the class on the surrounding element. the actual stroke always comes through as #000 in the DOM. Class `c20` is applying the correct color via CSS styling.

Reply:
Ah, yes. I used to know that. Thanks for the reminder. :)